src/main/webapp/WEB-INF/pages/inc/header.ftl

```diff
@@ -5,9 +5,11 @@
 [#setting locale="en"]
 [#setting url_escaping_charset="UTF-8"]
 <!DOCTYPE html>
-<html xmlns="http://www.w3.org/1999/xhtml"
-      xmlns:dc="http://purl.org/dc/elements/1.1/"
-      xmlns:foaf="http://xmlns.com/foaf/0.1/">
+<html prefix="dct: http://purl.org/dc/terms/
```

Comment:
Here you changed from dublin core elements to terms. Please note that for flexibility, dc elements was chosen on purpose because their values can be free text and facilitated widespread use. 

The freemarker template that displays the resource homepage ([resource_new.ftl](https://github.com/gbif/ipt/blob/master/src/main/webapp/WEB-INF/pages/portal/resource_new.ftl)) uses this header.ftl file and contains RDFa. Therefore I am hoping you can also review this pages' RDFa to make sure it is also valid with respect to your changes? 

Given that dcterms must always be given by a URI, we could at least use dcterms for the DOI identifier and License URL.  

Anyways, it would be great to have your continued help on this. In case it helps, here is a very complete resource homepage: view-source:http://ipt.ala.org.au/resource?r=global Thanks again for this pull request @pietercolpaert